rust/processor/src/models/fungible_asset_models/v2_fungible_metadata.rs

```diff
@@ -13,12 +13,12 @@ use crate::{
         token_v2_models::v2_token_utils::TokenStandard,
     },
     schema::fungible_asset_metadata,
-    utils::{database::PgPoolConnection, util::standardize_address},
+    utils::util::standardize_address,
 };
 use ahash::AHashMap;
 use aptos_protos::transaction::v1::WriteResource;
+use bigdecimal::BigDecimal;
```

Comment:
why decimal vs int?

Reply:
usually for on-chain data, it's represented with BigDecimal.